GettingStartedwithAnsibleusingGNS3.md

```diff
@@ -0,0 +1,283 @@
+# EXOS: Getting Started with Ansible using GNS3
+
+## Sections:
+* [Pre-requisites](#pre-requisites)
+* [Setting up GNS3 with EXOS-VM](#setting-up-gns3-with-exos-vm)
+* [Setting up GNS3 with Network Automation Docker Container](#setting-up-gns3-with-network-automation-docker-container)
+* [Setting up a Test Topology](#setting-up-a-test-topology)
+	- [EXOS Device Setup](#exos-device-setup)
+	- [Network Automation Container Setup](#network-automation-container-setup)
+* [System Setup](#system-setup)
+	- [Installing Ansible](#installing-ansible)
+	- [Setting up the Virtual Environment](#setting-up-the-virtual-environment)
+	- [Configuring the Local hosts file](#configuring-the-local-hosts-file)
+	- [Creating an Ansible configuration file](#creating-an-ansible-configuration-file)
+	- [Creating and configuring the Ansible inventory file](#creating-and-configuring-the-ansible-inventory-file)
+	- [Creating and configuring the Variable file](#creating-and-configuring-the-variable-file)
+* [Testing the Setup](#testing-the-setup)
+
+
+## Pre-requisites
+
+- GNS3 installed on any supported OS
+- GNS3 VM installed on any supported Hypervisor
+- Virtual EXOS Image
+
+## Setting up GNS3 with EXOS-VM
+
+1. Download the EXOS appliance template from [GNS3 Marketplace]
+2. Download the required qcow2 file from our [GitHub Website]
+3. Import the downloaded template in GNS3 and load the qcow2 file downloaded in step 2. Visit this [link] for details
+4. Drag the installed appliance to the workspace and start it up
+5. Run few show commands on the device to make sure that its working fine
+>NOTE: Default credentials to login to the device is "admin"/"". Password is blank
+
+## Setting up GNS3 with Network Automation Docker Container
+
+1. Download the Network Automation template from [GNS3 Marketplace]
+2. Import the downloaded template in GNS3 and perform the basic installation
+3. Once done, the "Network Automation" Container can be seen under the "End Devices" list
+4. Drag the docker container to the workspace and GNS3 will automatically pull the required files from its repository over the Internet
+
+> By default, the Network Automation docker container includes Python 2.7, NAPALM, pyntc, Netmiko and Ansible. So no need to install those separately.
```

Comment:
Reword "So no need to install those separately." as "You do not need to install these separately."

Reply:
done